backend/routes/post.js

```diff
@@ -1,11 +1,11 @@
-var express = require('express')
-var router = express.Router()
+let express = require('express')
```

Comment:
If you really one to be a cool kid who writes functional js most your `let` should be `const`

Reply:
Normally I would avoid changing `let` to `var` because it is a lot of line changes that could cause nastly merge conflicts